source/packages/modulo_core/tests/testMovingReferenceFrame.cpp

```diff
@@ -1,202 +1,173 @@
-#include "modulo_core/Cell.hpp"
-#include "state_representation/Space/Cartesian/CartesianState.hpp"
-#include "state_representation/Space/Cartesian/CartesianPose.hpp"
-#include "state_representation/Space/Cartesian/CartesianTwist.hpp"
 #include "dynamical_systems/Circular.hpp"
```

Comment:
If you want to be very consistent, the state_rep and DS includes should now be treated as external libs (with `<>` and after the local header includes)

Reply:
that is a good point yes that should be there as well. I will change that